static/app/views/dashboards/widgetBuilder/widgetBuilderSortBy.spec.tsx

```diff
@@ -969,7 +969,7 @@ describe('WidgetBuilder', function () {
       await selectEvent.openMenu(screen.getAllByText('count(\u2026)')[1]!);
 
       // 12 options in the dropdown
```

Comment:
Nit: maybe it's better to just remove this comment as well. It doesn't really add anything to the understanding of the next line